Game/Source/GDKTestGyms/GDKTestGymsGameInstance.cpp

```diff
@@ -15,6 +16,7 @@ void UGDKTestGymsGameInstance::Init()
 	TickDelegateHandle = FTicker::GetCoreTicker().AddTicker(TickDelegate);
 	TickWindowTotal = 0;
 	GetEngine()->NetworkFailureEvent.AddUObject(this, &UGDKTestGymsGameInstance::NetworkFailureEventCallback);
+	NFRConstants::Get().Init(GetWorld());
```

Comment:
I like aspects of this - but I feel it might fall down when testing in editor. Thoughts?

Reply:
Yeah possibly, maybe its worth creating an actor for this like we do for latency tracer

Reply:
On second thoughts, changing worker flags in the editor will for sure break this